### PR TITLE
ATLTitle

### PR DESCRIPTION
**Pull request by Isasprengel**
simulator screen shot - iphone xr - 2019-02-21 at 13 27 33

### CODE REVIEW

File: Example/atlantica-ios/MoleculeCellViewController.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ class MoleculeCellViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        table?.register(ATLListDivider.self, forCellReuseIdentifier: "Cell")
+        table?.register(ATLTitleCell.self, forCellReuseIdentifier: "Cell")
```
Comment (kyonaganawa) [inline Example/atlantica-ios/MoleculeCellViewController.swift]: Let's create one example view controller for each Molecule? That way we don't have to keep changing this file.

File: atlantica-ios/Classes/Molecules/ATLListDivider/ATLListDivider.swift [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-public enum ATLListDividerVariations {
+public enum ATLListDividerTypes {
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLListDivider/ATLListDivider.swift]: ty 🔥

File: atlantica-ios/Classes/Molecules/ATLTitle/ATLTitle.swift [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+//  ATLTitle.swift
+//  atlantica-ios
+//
+//  Created by Outsmart on 21/02/2019.
+//
+
+import UIKit
+
+public enum ATLTitleTypes {
+    case normal
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLTitle/ATLTitle.swift]: This is the issue we were talking about earlier right? Can we try another aproach before closing this PR?

File: atlantica-ios/Classes/Molecules/ATLTitle/ATLTitle.swift [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        let m = ATLTitle.init(frame: CGRect.init(x: 0, y: 0, width: frame.size.width, height: frame.size.height))
+        m.fillSuperviewWithConstraints(superview: contentView)
+        molecule = m
+    }
+}
+
+    //MARK: - ATLTitleViewData
+
+public class ATLTitleViewData {
+    public var text: String?
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLTitle/ATLTitle.swift]: I was thinking about adding id to all molecule's view data. What do you think? Not sure if it's needed here.

File: atlantica-ios/Classes/Molecules/ATLTitle/ATLTitle.swift [hunk 5]
```diff
@@ -7,11 +7,14 @@
...
 import UIKit
 
-public enum ATLTitleTypes {
-    case normal
+public enum ATLTitleColorsTypes {
+    case basic
     case complementary
+}
+
+public enum ATLTitleFunctionsTypes {
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLTitle/ATLTitle.swift]: Nice! 👍

File: Example/atlantica-ios/MoleculeCellViewController.swift (hunk 1 above)
Comment (Isasprengel) [reply Example/atlantica-ios/MoleculeCellViewController.swift]: Check!

**Pull request merged**